### PR TITLE
Move data access dashboard button next to study title

### PR DESCRIPTION
**Pull request by chowington**
Closes https://github.com/VEuPathDB/web-monorepo/issues/567

### CODE REVIEW

File: packages/libs/eda/src/lib/workspace/EDAWorkspaceHeading.tsx [hunk 1]
```diff
@@ -62,50 +64,68 @@ export function EDAWorkspaceHeading({
...
+              }}
+            >
+              <div>
+                <FloatingButton
+                  themeRole="primary"
+                  text="New analysis"
+                  tooltip="Create a new analysis"
+                  textTransform="capitalize"
+                  size="medium"
+                  // @ts-ignore
```
Comment (dmfalke) [inline packages/libs/eda/src/lib/workspace/EDAWorkspaceHeading.tsx:99]: Can this be removed?

**Review by dmfalke — COMMENTED** [review]
This looks pretty good, but I think the button text is too big, and it should match the site theme. Maybe use a coreui button for this?

image

File: packages/libs/eda/src/lib/workspace/EDAWorkspaceHeading.tsx (hunk 1 above)
Comment (chowington) [reply packages/libs/eda/src/lib/workspace/EDAWorkspaceHeading.tsx:99]: Ah, yep!

**Comment by chowington** [comment]
@dmfalke I was thinking we might want to retain the styling of the button (at least for now) since it's moving to a new place and won't be where users are used to it. What do you think? Maybe just shrink the text a bit?

**Comment by dmfalke** [comment]
I definitely think we should shrink the font size. I suspect folks will also want it to match the rest of the button styles, but I also see your point.

@danicahelb @aurreco-uga do you have an opinion?

**Comment by dmfalke** [comment]
@chowington after some intense reflection, I think you're right.

**Comment by asizemore** [comment]
Just to keep it exciting, i'll disagree. My proposal is to use the Filled version of the coreui button and the theme color. The biggest reason is because what do we do the next time we need to make something more prominent? We'll have to make it more prominent than a button that's already broken the site style, so we'll have to break the site style bigger next time. This is a downhill road.

Now remember i am so far not at all involved in this ticket and we had already accepted the above proposal so please feel free to ignore this 😄

**Comment by chowington** [comment]
@asizemore The idea was to only keep the prominent style for a little while so users could find it at its new location; however I think using CoreUI's FilledButton works just as well. I've updated the style, as well as added an info box at the button's previous location saying it's been moved. What do y'all think?

image

**Comment by dmfalke** [comment]
I think this looks great. What do you think about adding a little more space (doubling, maybe) between the study name and the button?

**Comment by asizemore** [comment]
Looks great! I agree with @dmfalke that a little more spacing would help. Probably just enough so that the button is about one "space" (like the space between words) from the end of the study name?

**Comment by chowington** [comment]
How about this?

image

**Comment by danicahelb** [comment]
Looks great @chowington! 

Please change the verbiage to `Study management dashboard` or more simply `Management dashboard`

**Review by dmfalke — APPROVED** [review]
Changes look great. Pending changes related to Danica's request, feel free to merge.

**Pull request merged**